src/pagination/pagination.ts

```diff
@@ -68,6 +78,19 @@ export class NgbPagination implements OnChanges {
   @Input() ellipses: boolean;
 
   /**
+   * Text to display for the first page button.
+   */
+  @Input()
+  set firstText(value: string) {
+    this._firstText = value;
+    if (value !== this._defaultConfig.firstText) {
+      this._firstTextSr = value;
```

Comment:
Why the difference between the screenreader text and the regular text? Why not just use the regular text?